packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -162,8 +167,10 @@ function _generateTableProps(jsonDocs: IPageJson | undefined): IApiReferencesTab
       });
     }
 
-    // to ensure that I{componentName}Props comes first
-    if (kind === 'interface' && propsName === table.name) {
+    if (jsonDocs.group === 'references' && jsonDocs.name === name) {
+      results.unshift(tableProps);
+    } else if (kind === 'interface' && propsName === name) {
```

Comment:
I'm very surprised we weren't using `name` instead of `table.name` before given that we had already extracted it from above.